code/modules/client/preferences_savefile.dm

```diff
@@ -691,6 +693,8 @@
 	undershirt = sanitize_inlist(undershirt, gender == MALE ? GLOB.undershirt_m : GLOB.undershirt_f, initial(undershirt))
 	backbag = sanitize_integer(backbag, 1, length(GLOB.backbaglist), initial(backbag))
 	preferred_armor = sanitize_inlist(preferred_armor, GLOB.armor_style_list, "Random")
+	if(!night_vision_color) night_vision_color = NV_COLOR_GREEN
```

Comment:
no inline ifs

Reply:
i just deleted it . i realize is not necesary anyways